<!DOCTYPE html>
<html><head><title>AssertEx</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">9 references to AssertEx</div><div class="rA">Assembly-CSharp (9)</div><div class="rG" id="Assembly-CSharp"><div class="rF"><div class="rN">Assets\UnitTests\UnitTests.cs (9)</div>
<a href="../Assets/UnitTests/UnitTests.cs.html#337"><b>337</b><i>AssertEx</i>.Throws&lt;InvalidOperationException&gt;(() =&gt; Observable.Empty&lt;int&gt;().Aggregate((x, y) =&gt; x + y).Wait());</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#986"><b>986</b><i>AssertEx</i>.Catch&lt;InvalidOperationException&gt;(() =&gt; d.Disposable = id2);</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#4303"><b>4303</b><i>AssertEx</i>.Catch&lt;TimeoutException&gt;(() =&gt;</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#6097"><b>6097</b><i>AssertEx</i>.Throws&lt;ArgumentOutOfRangeException&gt;(() =&gt; range.Skip(-1));</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#7727"><b>7727</b><i>AssertEx</i>.Throws&lt;ArgumentOutOfRangeException&gt;(() =&gt; Observable.Range(1, -1).ToArray().Wait());</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#9411"><b>9411</b><i>AssertEx</i>.Throws&lt;ObjectDisposedException&gt;(() =&gt; subject.OnNext(0));</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#9412"><b>9412</b><i>AssertEx</i>.Throws&lt;ObjectDisposedException&gt;(() =&gt; subject.OnError(new Exception()));</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#9413"><b>9413</b><i>AssertEx</i>.Throws&lt;ObjectDisposedException&gt;(() =&gt; subject.OnCompleted());</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#9461"><b>9461</b><i>AssertEx</i>.Throws&lt;ArgumentOutOfRangeException&gt;(() =&gt; range.Take(-1));</a>
</div>
</div>
</body></html>